<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>notes</class>
 <widget class="QDialog" name="notes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>402</width>
    <height>335</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Notes</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>221</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTextEdit" name="noteEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="saveNoteBttn">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>240</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="newNoteBttn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>New Note</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>300</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>newNoteBttn</sender>
   <signal>clicked()</signal>
   <receiver>notes</receiver>
   <slot>on_newNoteBttn_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>47</x>
     <y>251</y>
    </hint>
    <hint type="destinationlabel">
     <x>200</x>
     <y>167</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saveNoteBttn</sender>
   <signal>clicked()</signal>
   <receiver>notes</receiver>
   <slot>on_saveNoteBttn_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>357</x>
     <y>251</y>
    </hint>
    <hint type="destinationlabel">
     <x>200</x>
     <y>167</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>exitButton</sender>
   <signal>clicked()</signal>
   <receiver>notes</receiver>
   <slot>on_exitButton_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>357</x>
     <y>321</y>
    </hint>
    <hint type="destinationlabel">
     <x>200</x>
     <y>167</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_saveNoteBttn_clicked()</slot>
  <slot>on_newNoteBttn_clicked()</slot>
  <slot>on_exitButton_clicked()</slot>
 </slots>
</ui>
